config/environments/production.rb

```diff
@@ -29,7 +29,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
Can we default this to false, but allow it to be set by an environment variable?